### DISCUSSION TITLE
Change Label in Create issue is not available

### DISCUSSION THREAD

**Discussion by barchakuz** [camicroscope/caMicroscope#746] [Q&A]
Hello there, could someone help to change labels to newly created issues, because i want to create issues for enhancement purpose but label it don't show label option and after submitting issue, it automatically converted into bug label.
I have tried searching on internet but all suggesting using edit option to label which is not available in this case.
![tt](https://github.com/camicroscope/caMicroscope/assets/66128370/3d42a048-01cd-4b18-b676-08c011e49edf)

**Comment by birm** [comment]
That's interesting. I have to admit that I don't know why this is happening. If you want a label, you could just use brackets at the end, e.g. 'Important issue [ENH]' or something, or just mention it in the description text.
